examples/todo/ui/todo.slint

```diff
@@ -50,6 +53,10 @@ MainWindow := Window {
         { title: "Profit", checked: false },
     ];
 
+    property <bool> is-header-visible: false;
```

Comment:
Unlike "sort-by-name" this doesn't suffer from verb ambiguity - I suggest to rename this to simply "header-visible" or "show-header".